packages/text-sets/src/loadTextSets.ts

```diff
@@ -84,10 +88,10 @@ export default async function loadTextSets() {
   ];
 
   const results = await Promise.all(
-    textSets.map(async (name) => {
+    textSets.map(async (name: string): Promise<Array<string | TextSet[]>> => {
```

Comment:
Curious, doesn't TypeScript infer this type without having to define it manually?

Reply:
Ok, it seemed to infer it correctly when I tried locally but perhaps it was just a local thing.